### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team please review my code.
-added Wishlist page structure
-added CSS for the page

Go on to the heart icon to see the actual page

Live preview: https://stylevilla-wishlist.netlify.app/

### CODE REVIEW

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    <!-- Main content -->
+    <h1 class="heading"><span> My Wishlist </span></h1>
+    <div class="product-container">
+      <div class="product-card">
+        <div class="img-container">
+          <img src="/image/card.jpg" alt="image" />
+        </div>
+        <div class="info">
+          <div class="title">Women Hoddy</div>
+          <div class="price">500</div>
```
Comment (HarshnaNHaswani) [inline wishlist.html:50]: use span/ p instead

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  <body>
+    <header class="header">
+      <div class="header-1">
+        <a href="index.html" class="logo">
+          <i class="fa-solid fa-bag-shopping"></i> StyleVilla
+        </a>
+        <form action="" class="search-form">
+          <input
+            type="search"
+            name=""
```
Comment (HarshnaNHaswani) [inline wishlist.html]: empty name?

**Review by HarshnaNHaswani — COMMENTED** [review]
Looks good to me, just add a little more spacing between nav icons in the site header,
also add an option to remove items from wish list

**Review by chanchal-panpaliya — APPROVED** [review]
UI and code looks good to merge .

File: wishlist.html (hunk 1 above)
Comment (sahiba0915) [reply wishlist.html:50]: done

File: wishlist.html (hunk 2 above)
Comment (sahiba0915) [reply wishlist.html]: done

**Pull request merged**